apps/server/src/treematch/match-math.ts

```
@@ -0,0 +1,78 @@
+// Unit arithmetic for TreeMatch. TTC works in centi-units (100 = 1 tree, so
+// partial trees are representable); the web works in whole trees. Everything
+// stored in treematch_allocation is centi-units, and conversion happens only
+// here, at the API boundary.
+//
+// Kept pure and dependency-free so the arithmetic is unit-testable without a
+// database or the Nest container.
+
+export const CENTI = 100;
```

ESLint

[error] Delete `;`

(prettier/prettier)